tokens-studio/figma-plugin
# Fix styling issues post 2.0 release

Addresses #3088 - the themes that are now selected in the styles export modal shall be exported as styles.
Resolves #3083  - the 2nd point when multiple themes are enabled, then the styles don't show in Figma

# Description

- When we select the themes in the export modal to be exported as styles, the expectation now would be to export those themes regardless of their active/inactive status.
- When multiple themes are enabled, especially core/global, figma ceased to display the corresponding style token applied. This PR should address that

## Type of change


# How to test this

1st Issue:

* Create some themes in the plugin.
* Keep them inactive.
* Go to Export Styles&Variables, choose the themes you wish to be exported as styles
* They should be exported irrespective of them being inactive

2nd Issue:

- Create some themes, along with a core/global theme
- Apply light/dark theme, theme switching should work fine, along with the correct styles being applied(and displayed on Figma) without core theme enabled
- As soon you apply core theme, the theme switching should work fine between light/dark or any other themes, along with the pertinent styles being showed in Figma

# Screenshots or video (if necessary):
